src/main/java/buildcraftAdditions/client/models/ModelKineticDuster.java

```diff
@@ -17,13 +18,10 @@
  */
 public class ModelKineticDuster extends ModelBiped {
 
-	public static ModelKineticDuster INSTANCE = new ModelKineticDuster();
-	public static ModelKineticDuster INSTANCE2 = new ModelKineticDuster();
-
-	private final ResourceLocation TEXTURE0 = new ResourceLocation("bcadditions", "textures/models/duster3-texturemap0.png");
-	private final ResourceLocation TEXTURE1 = new ResourceLocation("bcadditions", "textures/models/duster3-texturemap1.png");
-	private final ResourceLocation TEXTURE2 = new ResourceLocation("bcadditions", "textures/models/duster3-texturemap2.png");
-	private final ResourceLocation TEXTURE3 = new ResourceLocation("bcadditions", "textures/models/duster3-texturemap3.png");
+	private static final ResourceLocation TEXTURE0 = new ResourceLocation("bcadditions", "textures/models/duster3-texturemap0.png");
+	private static final ResourceLocation TEXTURE1 = new ResourceLocation("bcadditions", "textures/models/duster3-texturemap1.png");
+	private static final ResourceLocation TEXTURE2 = new ResourceLocation("bcadditions", "textures/models/duster3-texturemap2.png");
+	private static final ResourceLocation TEXTURE3 = new ResourceLocation("bcadditions", "textures/models/duster3-texturemap3.png");
```

Comment:
why would you make these static? it serves no purpose especially since they are private